scripts/chef-automate-install.sh

```diff
@@ -181,21 +198,34 @@ _deployAutomateV2() {
 	if [[ ! -e "automate-credentials.toml" ]]; then
 		info "Deploying Automate V2"
 		export GRPC_GO_LOG_SEVERITY_LEVEL=info GRPC_GO_LOG_VERBOSITY_LEVEL=2
-		./chef-automate deploy config.toml --accept-terms-and-mlsa --debug
+		./chef-automate deploy config-public.toml --accept-terms-and-mlsa --debug
 	else
 		info "Automate V2 already deployed"
 	fi
 	)
 }
 
+_uploadChefAutomatePasswordToAzureKeyVault() {
+	(
+		info "Uploading automate credentials to key vault"
+		cd "${DELIVERY_DIR}"
```

Comment:
This will work fine in this case but generally I would always advise caution when changing directory without later changing back.  Could add a `cd -` or do `pushd/popd` to ensure subsequent commands aren't affected.